ui/components/NodeInputHidden.tsx

```diff
@@ -1,7 +1,35 @@
 import { NodeInputProps } from "./helpers";
 import React, { FC } from "react";
+import { Input } from "@canonical/react-components";
+
+export const NodeInputHidden: FC<NodeInputProps> = ({
+  node,
+  attributes,
+  setValue,
+}) => {
+  if (node.meta.label?.text === "Change e-mail") {
+    return (
+      <Input
+        type="email"
+        labelClassName="password-label"
+        label={
+          <>
+            <span>E-Mail</span>
+            <a
+              href={`${window.location.href}&reset_email=1`}
```

Comment:
question: This causes the page to reload and adds this on the browser history, can't we avoid it?